html_xblock/html.py

```diff
@@ -89,7 +92,9 @@ def studio_view(self, context=None):  # pylint: disable=unused-argument
 
         js_data = {
             'editor': self.editor,
-            'skin_url': self.runtime.local_resource_url(self, 'public/skin'),
```

Comment:
@tecoholic, I believe it's fine to replace them - we'll be delivering these through CDNs, so it shouldn't be a high cost for improving the maintainability. Especially since these scripts are mainly used in Studio, so it does not affect learners.

Reply:
Cool. Makes sense. I have replaced them in 1f55ba1.